### PR TITLE
Update to graalvm and mandrel 22.3.0 - 2.13

### LINKED ISSUE #26830: Lost context in @Blocking gRPC Service behind a GlobalInterceptor
I found that Quarkus Server register `io.quarkus.grpc.runtime.supports.blocking.BlockingServerInterceptor` for each blocking gRPC service with the `@Blocking` annotation before it starts.

If some global interceptors (with `@GlobalInterceptor`) set the Context value will not work. The `Context` will still be empty in the actual rpc method handler.

Is it a normal phenomenon because `Context` should be placed after `BlockingServerInterceptor` 's `vert.x.executeBlocking`? There are some associated issues: https://github.com/quarkusio/quarkus/issues/14665, https://github.com/quarkusio/quarkus/issues/13959.

Finally, I replace the `@GlobalInterceptor` with `@RegisterInterceptor(MyInterceptor.class)`, and everything is to be ok which seems to prove my point.

I don't particularly understand the technical details of it, and these are what I got from trying and debugging. I've been only using Spring and Micronaut before.

origin question:

https://stackoverflow.com/questions/73039706/lost-context-in-blocking-grpc-service-with-quarkus-behind-a-globalinterceptor

### LINKED ISSUE #28292: `StorkClientRequestFilter` not registered when using `ClientBuilder` to build a jax-rs client
### Describe the bug

When using Stork with a manually-created jax-rs client (via `ClientBuilder.newClient()`), the `StorkClientRequestFilter` is not automatically registered.

Stork worked fine with the jax-rs client in Quarkus 2.12, but in 2.13 this is broken unless the user manually registers the filter.

See https://github.com/quarkusio/quarkus-super-heroes/blob/main/rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/client/VillainClient.java#L38

If the user does not manually register the filter then they will end up with a failure that looks something like

```
java.lang.IllegalArgumentException: Invalid REST Client URL used: 'stork://villain-service/api/villains/random'
```

At a very minimum, this should be documented in the Quarkus 2.13 release notes.

### Expected behavior

I would expect Stork to "just work" with the jax-rs client, as it did prior to Quarkus 2.12.

### Actual behavior

The application fails if the user does not manually register the filter.

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

The `OpenTelemetryClientFilter` seems to be registered automatically when creating the jax-rs client this way, so there must be a way to do this as well for the `StorkClientRequestFilter`.

### PR DESCRIPTION
**Pull request by cescoffier**
- Update to GraalVM and Mandrel 22.3.0 images
- Use the GraalVM 22.3 AlwaysInline annotation

### CODE REVIEW

File: docs/src/main/asciidoc/building-native-image.adoc [hunk 1]
```diff
@@ -653,10 +653,10 @@ Building fully statically linked binaries enables the usage of a https://hub.doc
 
 Sample multistage Dockerfile for building an image from `scratch`:
 
-[source, dockerfile]
+[source,dockerfile,subs=attributes+]
 ----
 ## Stage 1 : build with maven builder image with native capabilities
-FROM quay.io/quarkus/ubi-quarkus-native-image:22.0-java11 AS build
+FROM quay.io/quarkus/ubi-quarkus-graalvmce-builder-image:{graalvm-flavor} AS build
```
Comment (zakkak) [inline docs/src/main/asciidoc/building-native-image.adoc:659]:
```suggestion
FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} AS build
```

**Review by zakkak — DISMISSED** [review]
LGTM, other than the documentation issue. 

Thanks @cescoffier

**Comment by gsmet** [comment]
@ppalaga here is your 2.13 branch with GraalVM 22.3 ^

**Comment by gsmet** [comment]
Don't worry about the quickstarts compilation, that's normal.

File: docs/src/main/asciidoc/building-native-image.adoc (hunk 1 above)
Comment (cescoffier) [reply docs/src/main/asciidoc/building-native-image.adoc:659]: Oh! Nice catch!

**Review by gsmet — APPROVED** [review]

**Review by zakkak — APPROVED** [review]

**Pull request merged**